src/main/scala/io/iohk/ethereum/vm/ProgramContext.scala

```diff
@@ -6,40 +6,33 @@ import io.iohk.ethereum.domain._
 object ProgramContext {
   def apply[W <: WorldStateProxy[W, S], S <: Storage[S]](
     stx: SignedTransaction,
+    recipientAddress: Address,
```

Comment:
to pass in contract that is created elsewhere if we have contract creation transaction,
to remove ether transfer from here i have to create this contract elsewhere, do ether transfer and then pass address to ProgramContext